src/components/ButtonForm/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+
+import styles from './ButtonForm.module.scss';
+
+function ButtonForm({ type, isFilled, text }) {
+  const { buttonFill, button } = styles;
+
+  /* eslint-disable react/button-has-type */
+  return (
+    <div>
+      <button type={type} className={clsx(button, isFilled && buttonFill)}>
```

Comment:
what is `isFilled`?
also when using conditionals on clsx use like an object:

`clsx(styles.button, { [styles.buttonFilled]: isFilled })`